bazel/external/proxy_wasm_cpp_host.BUILD

```diff
@@ -0,0 +1,37 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "include",
+    hdrs = glob(["include/proxy-wasm/**/*.h"]),
+    deps = [
+        "@proxy_wasm_cpp_sdk//:common_lib",
+    ],
+)
+
+cc_library(
+    name = "lib",
+    srcs = glob(
+        [
+            "src/**/*.h",
+            "src/**/*.cc",
+        ],
+        exclude = ["src/**/wavm*"],
+    ),
+    copts = ["-std=c++14"],
+    deps = [
+        ":include",
+        "//external:abseil_flat_hash_map",
+        "//external:abseil_optional",
+        "//external:abseil_strings",
+        "//external:protobuf",
+        "//external:wee8",
+        "//external:zlib",
+        "//external:ssl",
```

Comment:
Nit: This should be sorted alphabetically, CI will probably complain about it using `buildifier` anyway.

Reply:
Done